builtin/providers/aws/resource_aws_route53_record.go

```
@@ -184,7 +184,10 @@ func resourceAwsRoute53RecordRead(d *schema.ResourceData, meta interface{}) erro
 
 		found = true
 
-		d.Set("records", record.ResourceRecords)
+		err := d.Set("records", flattenResourceRecords(record.ResourceRecords))
+		if err != nil {
+			log.Printf("[DEBUG] Error setting records for: %s, error: %#v", en, err)
```

Should we return this error instead of just logging it?